Electron-Basic-Screen-Sharing/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "electron-basic-screen-sharing-v17",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "start": "electron ."
+  },
+  "contributors": [
+    "Manik Sachdeva",
+    "German Goldenstein",
+    "Christian Pettet"
+  ],
+  "license": "ISC",
+  "dependencies": {
+    "electron": "^17.0.0-alpha.3"
```

Comment:
Not sure if possible, given we're using the (now) beta version of electron, but something like "Electron 17+" would be sweet.

Reply:
I can update it to Electron 17, npm has been giving me errors since there's no official 17 release.